src/pb_stub.cc

```diff
@@ -1824,11 +1824,13 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
   py::class_<Metric, std::shared_ptr<Metric>>(module, "Metric")
       .def("increment", &Metric::SendIncrementRequest)
       .def("set", &Metric::SendSetValueRequest)
+      .def("observe", &Metric::SendObserveRequest)
       .def("value", &Metric::SendGetValueRequest);
 
   py::enum_<MetricKind>(module, "MetricKind")
       .value("COUNTER", MetricKind::kCounter)
       .value("GAUGE", MetricKind::kGauge)
+      .value("HISTOGRAM", MetricKind::kHistogram)
```

Comment:
+1 on @oandreeva-nv comment for testing in python backend too:

Can add to the existing custom metrics tests:
- https://github.com/triton-inference-server/server/blob/main/qa/L0_backend_python/custom_metrics/test.sh
- https://github.com/triton-inference-server/server/blob/main/qa/python_models/custom_metrics/model.py

Reply:
Tests added https://github.com/triton-inference-server/server/pull/7525